Realm/RLMRealm.h

```diff
@@ -815,6 +815,16 @@ NS_REFINED_FOR_SWIFT;
  */
 - (struct RLMClassPrivileges)privilegesForClassNamed:(NSString *)className;
 
+#pragma mark - Utilities
+
+/**
+ Returns the current active number of versions of a Realm instance.
```

Comment:
I don't think users will have any clue what this means and it isn't really accurate. "instance" in particular makes no sense here; we usually use that to refer to RLMRealm objects and that's not applicable to this.

Reply:
would simply saying ' Returns the number of active versions of this Realm.' suffice?